rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -824,6 +828,81 @@ void getFederationElection_whenElectionIsNotInStorage_shouldReturnDefault() {
 
     }
 
+    @Test
+    void getNextFederationCreationBlockHeight_preIris300_storageIsNotAccessedAndReturnsEmpty() {
+
+        // Arrange
+
+        ActivationConfig.ForBlock activations = ActivationConfigsForTest.papyrus200().forBlock(0L);
+
+        StorageAccessor storageAccessor = new InMemoryStorage();
+        // Putting some value in the storage just to then assert that before fork, the storage won't be accessed.
+        storageAccessor.saveToRepository(NEXT_FEDERATION_CREATION_BLOCK_HEIGHT_KEY.getKey(), RLP.encodeBigInteger(
+            BigInteger.valueOf(1_300_000)));
+
+        // Act
+
+        FederationStorageProvider federationStorageProvider = new FederationStorageProviderImpl(storageAccessor);
+
+        // Assert
+        assertEquals(Optional.empty(), federationStorageProvider.getNextFederationCreationBlockHeight(activations));
+
+    }
+
+    @Test
+    void getNextFederationCreationBlockHeight_postIris300_getsValueFromStorage() {
```

Comment:
```suggestion
    void getNextFederationCreationBlockHeight_whenIrisIsActive_getsValueFromStorage() {
```

wdyt?

Reply:
No big deal. I could go with any of them. Let's discuss with the rest of the team to see which one is more convenient as a standard.